Browser/CefSharpBrowser/CefOp/CustomRequestHandler.cs

```diff
@@ -12,12 +14,15 @@ public class CustomRequestHandler : RequestHandler
 
 	private bool PixiSettingEnabled { get; }
 	private bool UseGadgetRedirect { get; }
+	private GadgetServerOptions GadgetBypassServer { get; }
+	private string GadgetBypassServerCustom {get; }
 
-
-	public CustomRequestHandler(bool pixiSettingEnabled, bool useGadgetRedirect)
+	public CustomRequestHandler(bool pixiSettingEnabled, bool useGadgetRedirect,GadgetServerOptions gadgetBypassServer, string gadgetBypassServerCustom)
```

Comment:
https://stackoverflow.com/questions/5755942/how-do-you-auto-format-code-in-visual-studio